.github/workflows/protocol-release.yml

```
@@ -120,6 +120,9 @@ jobs:
           TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
         run: |
           twine upload -r pypi dist/*
+        with:
```

actionlint (1.7.3)

this step is for running shell command since it contains at least one of "run", "shell" keys, but also contains "with" key which is used for running action

(syntax-check)